### PR TITLE
Update citation section

### LINKED ISSUE #1297: Documentation: update citation guidelines
The RTD documentation only mentions the article citation, and the README offers only Zenodo DOIs for the software (as opposed to the published article).

We should make it easy for others to cite the software itself, as well as the article discussing it, by providing a full example citation and cut-and-pasteable BibTeX entry.

### PR DESCRIPTION
**Pull request by brenthuisman**
* Add Zenodo citation snippet.
* Correct Nora's last name, which is Abi Akar.
* Small css fix for change in docutils, see: https://github.com/readthedocs/sphinx_rtd_theme/issues/1115

Fixes #1297.

### CODE REVIEW

File: doc/index.rst [hunk 1]
```diff
@@ -33,17 +33,64 @@ Documentation organisation
...
+
+Version 0.1
+    |z01|
+
+Introductory paper
+    .. code-block:: latex
+
+        @INPROCEEDINGS{
+            paper:arbor2019,
+            author={N. A. {Akar} and B. {Cumming} and V. {Karakasis} and A. {Küsters} and W. {Klijn} and A. {Peyser} and S. {Yates}},
```
Comment (noraabiakar) [inline doc/index.rst:80]: My last name is "Abi Akar". The "Abi" often gets classified as a middle name.

File: doc/index.rst [hunk 2]
```diff
@@ -33,17 +33,64 @@ Documentation organisation
...
+Latest version
+    |zlatest|
+
+Version 0.5
+    |z05|
+
+    .. code-block:: latex
+
+        @software{nora_abi_akar_2021_4428108,
+        author       = {Nora Abi Akar and
```
Comment (thorstenhater) [inline doc/index.rst:47]: Do these render nicely in bibtex? I never remember where to put "" and {}.

**Review by thorstenhater — APPROVED** [review]
Looks good. Although some quoting might be helpful in bibtex entry.

File: doc/index.rst (hunk 2 above)
Comment (brenthuisman) [reply doc/index.rst]: Good question. It's a copy paste from Zenodo, I added the extra brackets for Nora's last name and verified that it renders well.

**Pull request merged**